containers/php/.devcontainer/devcontainer.json

```diff
@@ -2,33 +2,37 @@
 	"name": "PHP",
 	"build": {
 		"dockerfile": "Dockerfile",
-		"args": { 
+		"args": {
 			// Update VARIANT to pick a PHP version: 8, 8.1, 8.0, 7, 7.4
 			// Append -bullseye or -buster to pin to an OS version.
 			// Use -bullseye variants on local on arm64/Apple Silicon.
 			"VARIANT": "8.1-apache-bullseye",
 			"NODE_VERSION": "lts/*"
 		}
 	},
-	
 	// Set *default* container specific settings.json values on container create.
-	"settings": { 
+	"settings": {
 		"php.validate.executablePath": "/usr/local/bin/php"
 	},
-
 	// Add the IDs of extensions you want installed when the container is created.
 	"extensions": [
 		"xdebug.php-debug",
 		"bmewburn.vscode-intelephense-client",
 		"mrmlnc.vscode-apache"
 	],
-
-	// Use 'forwardPorts' to make a list of ports inside the container available locally.
-	"forwardPorts": [8080],
-
+	// Use 'portsAttributes' to control auto forwarding of ports
+	"portsAttributes": {
+		"8080": {
+			"label": "Application",
+			"onAutoForward": "notify",
```

Comment:
There is a hanging comma here that is causing the CI to fail.

Reply:
Didn't know `JSON with Comments` was a relaxed mode for parsing JSON.  That was a new one for me.